src/AudioPasteDialog.cpp

```diff
@@ -0,0 +1,147 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file AudioPasteDialog.cpp
+
+  @author Vitaly Sverchinsky
+
+**********************************************************************/
+
+#include "AudioPasteDialog.h"
+
+#include <wx/sizer.h>
+#include <wx/checkbox.h>
+#include <wx/stattext.h>
+#include <wx/radiobut.h>
+
+#include "ShuttleGui.h"
+#include "prefs/TracksBehaviorsPrefs.h"
+#include "widgets/WindowAccessible.h"
+
+BEGIN_EVENT_TABLE(AudioPasteDialog, wxDialogWrapper)
+   EVT_BUTTON(wxID_OK, AudioPasteDialog::OnContinue)
+   EVT_BUTTON(wxID_CANCEL, AudioPasteDialog::OnCancel)
+END_EVENT_TABLE()
+
+namespace {
+
+const TranslatableString audioPasteModeText[] = {
+   XO("Smart clip.\n"
+      "The entire source clip will be pasted into your project, allowing you to access\ntrimmed audio data anytime."),
```

Comment:
I assume all visible design and wording changes were prescribed by someone else, but I think "any time" is more proper English usage.  Although Grammarly just told me it's considered acceptable as one word now.  @LWinterberg

Reply:
Grammarly says:
>A century ago, it was standard to write any time as two words in all contexts. But it’s now perfectly acceptable to write anytime as one word when you’re using it as an [adverb](https://www.grammarly.com/blog/adverb/). However, some readers still consider it a casualism, so you may want to stick to the two-word version for extremely formal writing.

I'm minded to agree with @Paul-Licameli  the it would be better as "any time" with the space.